contribute/hacker-guide.md

```diff
@@ -100,47 +100,25 @@ If you are new to Git and branching, read the [Branching Chapter](http://git-scm
 
 The next step after cloning your fork is setting up your machine to build Scala.
 
-* It is recommended to use Java `1.6` (not `1.7` or `1.8`, because they might cause occasional glitches).
-* The build tool is `ant`. If you are behind a HTTP proxy, include [`ANT_ARGS=-autoproxy`](https://ant.apache.org/manual/proxy.html) in your environment.
-* The build runs the `pull-binary-libs.sh` script to download bootstrap libs. This requires `bash` and `curl`.
-* The majority of our team works on Linux and OS X, so these operating systems are guaranteed to work.
-* Windows is supported, but it might have issues. Please report to [the issue tracker](https://issues.scala-lang.org/) if you encounter any.
+You need the following tools:
 
-Building Scala is as easy as running `ant` in the root of your cloned repository. Be prepared to wait for a while-- a full "clean" build
-takes 8+ minutes depending on your machine (and up to 30 minutes on older machines with less memory). Incremental builds are usually within 30-120 seconds range (again, your mileage might vary
-with your hardware).
-
-    16:50 ~/Projects/scala (ticket/6725)$ ant
-    Buildfile: /Users/xeno_by/Projects/scala/build.xml
-
-    strap.clean:
-
-    pack.clean:
-
-    init.jars.check:
-
-    init.jars:
-         [echo] Updating bootstrap libs.  (To do this by hand, run ./pull-binary-libs.sh)
-         [exec] Resolving [943cd5c8802b2a3a64a010efb86ec19bac142e40/lib/ant/ant-contrib.jar]
-
-    ...
-
-    pack.bin:
-        [mkdir] Created dir: /Users/xeno_by/Projects/scala/build/pack/bin
+* A Java SDK. The baseline version is 6 for 2.11.x and 8 for 2.12.x. It's possible to use a later SDK for local development, but the continuous integration builds will verify against the baseline version.
+* `sbt`, an interactive build tool commonly used in Scala projects. Acquiring sbt manually is not necessary -- the recommended approach is to download the [sbt-extras runner script](https://github.com/paulp/sbt-extras/blob/master/sbt) and use it in place of `sbt`. The script will download and run the correct version of sbt when run from the Scala repository's root directory.
+* `curl` -- the build uses `curl` in the `pull-binary-libs.sh` script to download bootstrap libs.
 
-    pack.done:
+The majority of our team works on Linux and OS X, so these operating systems are guaranteed to work. Windows is supported, but it might have issues. Please report to [the issue tracker](https://issues.scala-lang.org/) if you encounter any.
 
-    build:
-
-    BUILD SUCCESSFUL
-    Total time: 9 minutes 41 seconds
+Building Scala is as easy as running `sbt dist/mkPack` in the root of your cloned repository. Be prepared to wait for a while -- a full "clean" build
+takes 8+ minutes depending on your machine (and up to 30 minutes on older machines with less memory). Incremental builds are usually within 30-120 seconds range (again, your mileage might vary
+with your hardware).
```

Comment:
A clean build with `sbt dist/mkPack` took me 264 seconds. I then modified src/compiler/scala/reflect/macros/compiler/Errors.scala and ran `sbt dist/mkPack` again. The second run took 21 seconds.